Solutions/Azure Activity/Package/createUiDefinition.json

```diff
@@ -88,7 +88,7 @@
             "name": "workbooks-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "The workbooks installed with the Azure Activity solution provide insights into operations on different Azure resources. After installing the solution, start using the workbook in Manage solution view."
```

Comment:
Hi @v-rbajaj please revert this change for the workbook text as it is specifically defined for each solution, thanks!